tests/utils/test_snuba_exception.py

```diff
@@ -0,0 +1,42 @@
+from snuba.utils.snuba_exception import SnubaException
+
+
+def test_exception() -> None:
+    class MyException(SnubaException):
+        pass
+
+    message = "I am an exception beep boop"
+    try:
+        raise MyException(message, extra="data")
+    except MyException as e:
+        assert e.message == message
+        assert e.extra_data["extra"] == "data"
+        edict = e.to_dict()
+        resurfaced_e = SnubaException.from_dict(edict)
+        assert isinstance(resurfaced_e, MyException)
+        assert resurfaced_e.message == message
+        assert resurfaced_e.extra_data == {"extra": "data"}
+        assert resurfaced_e.should_report
+
+
+def test_from_standard_exception() -> None:
+    message = "I am an exception beep boop"
+    snubs_exc = None
+    # We can create snuba exceptions from standard ones
+    try:
+        raise Exception(message)
+    except Exception as e:
+        snubs_exc = SnubaException.from_standard_exception_instance(e)
+        assert isinstance(snubs_exc, SnubaException)
+        assert snubs_exc.__class__.__name__ == "Exception"
+        assert snubs_exc.message == message
+        assert snubs_exc.extra_data["from_standard_exception"]
+        assert snubs_exc.should_report
+
+    # if we create a SnubaException from a standard exception
+    # before, we'll create the same one again
+    try:
+        raise Exception("Other message")
+    except Exception as e:
+        new_snubs_exc = SnubaException.from_standard_exception_instance(e)
```

Comment:
What is this testing exactly? Why would these be different, the only difference between them is the message you're passing in.

Reply:
This is testing the codepath of going from standard exception (e.g. AttributeError) to a SnubaException and that the message is preserved. The different messages are confusing so I'll fix that

Reply:
> Why would these be different, the only difference between them is the message you're passing in.

I just reread your question. Basically this is testing that any other time a bare `Exception` is raised, if we create a `SnubaException` from it, it will be the same type as the one raised on this machine before (even if the message is different)

Reply:
I think I understand, but this might be clearer if you used something other than a bare Exception, like a custom exception class or ValueError or something.